.github/workflows/push_pr.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         with:
           fetch-depth: 0
       - uses: helm/chart-testing-action@v2.2.1
+      - name: Add newrelic repository
+        run: helm repo add newrelic https://helm-charts.newrelic.com
```

Comment:
NIT/something to discuss: 

I would leave this as it is because there are more than two other repos that have the repository added like this BUT there is another (more cleaner if you ask me) way to add this that @sigilioso [has in its PR](https://github.com/newrelic/k8s-metadata-injection/blob/bb45f8a0c1d49fd47fa96bb95cd14b8603ce327f/.github/ct.yaml#L6-L7) but un the `ct.yaml` file.

We should decide which way we should standardize this. (In fact this is a job I have to do)

Reply:
I changed to follow the same approach for constancy, on the other hand it is making the add repo step a bit hidden for the unit test step ( i think that is also needed for that step if i'm not wrong)